styles/styles.css

```diff
@@ -0,0 +1,206 @@
+/* .container{
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr 1fr 3fr 50px;
+} */
+section {
+    border-bottom: 1px solid lightgray;
+}
+
+h2 {
+    font-size: 13px;
+    text-align: center;
+    font-weight: bold;
+    padding: 30px;
+}
+
+nav ul {
+    display: grid;
+    grid-template: 1fr 3fr / 60% 20% 20%;
+    font-size: 10px;
+    font-weight: bold;
+}
+
+ul {
+    list-style-type: none;
+}
+
+
+.about {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 50px;
+    }
+
+.about h1{
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 25px;
+}
+
+.about sub {
+    display: block;
+    font-weight: bold;
+    font-size: 15px;
+    margin-bottom: 1.5em;
+}
+.about a {
+    display: block;
+    color: black;
+    text-align: center;
+    font-size: 13px;
+    /* block-size: 30px; */
+    border: lightgray 1px solid;
+    background-color: white;
+    text-decoration: none;
+    border-radius: 5px;
+    padding: 8px 12px;
+
+}
+
+.about a:hover {
+    color: red;
+    background-color: aqua;
+}
+
+
+.quote {
+    text-align: justify;
+    font-style: italic;
+    line-height: 2; 
+    font-size: 13px;
+    padding: 40px 50px 40px 50px;
+    background-color: whitesmoke;
+
```

Comment:
The alignment of the quotes is a little off. The speaker appears to be a quote of its own.